blocks/shared-styles/_children/promo-image/styles.scss

```diff
@@ -1,3 +1,17 @@
 .promo-image {
   position: relative;
+
+  .promo-label {
+    .label {
+      color: #fff;
+      margin-left: 8px;
+      flex-grow: 0;
+      font-size: 16px;
```

Comment:
should this be font-size 12? and with height 12? not sure if I'm missing something 

looks like it came from here https://github.com/WPMedia/fusion-news-theme-blocks/pull/890/files#diff-249277bd5886fedffad4be3e3aa2d4254e03759db78db5c157ef65c1037093bbL35

Reply:
This has a pretty big visual impact, was this agreed with a designer?

<img width="519" alt="Microsoft Edge-2021-06-04-10-48 57" src="https://user-images.githubusercontent.com/868127/120783000-7fe1da80-c522-11eb-8393-55011c099661.png">

Reply:
i just updated these styles.

I will note to you and @JackHowa that i will need to update the padding and height for these, reason being because the icon and text alignment are off by a pixel and this is the only way to keep the label itself the same height while aligning the text and icon correctly.